### ISSUE TITLE
Support for hardware wallets

### ISSUE THREAD

**Issue by rene78** [ethereum/staking-deposit-cli#152]
Would be nice if the hardware wallet could be utilized to create withdrawal and signing keys instead of using the mnemonic generated in _eth2.0-deposit-cli_. As far as I know Ledger Nano X supports the BLS12-381 signature scheme.

**Comment by lekanovic** [comment]
Yeah I agree, using the ledger would feel much safer then having to keep the mnemonic seed around. Is this in the current roadmap?

**Comment by lekanovic** [comment]
Ledger has now released support:
https://www.ledger.com/blog/ledger-is-ready-for-ethereum-2/?utm_source=Twitter&utm_medium=Social&utm_campaign=Eth2Firmware

**Comment by DanielSel** [comment]
Are there any news for this?
